--- devel/flim/PLIST.orig
+++ devel/flim/PLIST
@@ -1,6 +1,6 @@
 @comment $NetBSD: PLIST,v 1.12 2023/09/03 14:58:38 mef Exp $
-info/mime-en.info
-info/mime-ja.info
+${EMACS_INFOPREFIX}/mime-en.info
+${EMACS_INFOPREFIX}/mime-ja.info
 ${EMACS_LISPPREFIX}/flim/eword-decode.el
 ${EMACS_LISPPREFIX}/flim/eword-decode.elc
 ${EMACS_LISPPREFIX}/flim/eword-encode.el
--- inputmethod/skk/PLIST.orig
+++ inputmethod/skk/PLIST
@@ -1,5 +1,5 @@
 @comment $NetBSD: PLIST,v 1.20 2020/08/24 20:40:00 ryoon Exp $
-info/skk.info
+${EMACS_INFOPREFIX}/skk.info
 ${EMACS_LISPPREFIX}/skk/ccc.el
 ${EMACS_LISPPREFIX}/skk/ccc.elc
 ${EMACS_LISPPREFIX}/skk/cdb.el
--- mail/mailutils/PLIST.orig
+++ mail/mailutils/PLIST
@@ -148,7 +148,7 @@
 include/mailutils/version.h
 include/mailutils/wordsplit.h
 include/mailutils/yyloc.h
-info/mailutils.info
+${EMACS_INFOPREFIX}/mailutils.info
 ${PLIST.guile}lib/libguile-mailutils-v-3.15.so
 lib/libmailutils.la
 lib/libmu_auth.la
--- mail/mew/PLIST.orig
+++ mail/mew/PLIST
@@ -8,7 +8,7 @@
 bin/mewest
 bin/mewl
 bin/smew
-info/mew.info
+${EMACS_INFOPREFIX}/mew.info
 man/man1/cmew.1
 man/man1/incm.1
 man/man1/mew-pinentry.1
--- mail/mu/PLIST.emacs.orig
+++ mail/mu/PLIST.emacs
@@ -1,5 +1,5 @@
 @comment $NetBSD: PLIST.emacs,v 1.3 2026/05/30 12:56:30 ryoon Exp $
-info/mu4e.info
+${EMACS_INFOPREFIX}/mu4e.info
 man/man1/mu-add.1
 man/man1/mu-cfind.1
 man/man1/mu-extract.1
--- mail/vm/PLIST.orig
+++ mail/vm/PLIST
@@ -3,8 +3,8 @@
 bin/base64-encode
 bin/qp-decode
 bin/qp-encode
-info/vm-pcrisis.info
-info/vm.info
+${EMACS_INFOPREFIX}/vm-pcrisis.info
+${EMACS_INFOPREFIX}/vm.info
 share/doc/vm/CHANGES
 share/doc/vm/COPYING
 share/doc/vm/NEWS
--- mail/wl-snapshot/PLIST.orig
+++ mail/wl-snapshot/PLIST
@@ -1,6 +1,6 @@
 @comment $NetBSD: PLIST,v 1.3 2023/09/16 11:24:17 mef Exp $
-info/wl-ja.info
-info/wl.info
+${EMACS_INFOPREFIX}/wl-ja.info
+${EMACS_INFOPREFIX}/wl.info
 share/doc/wl/en/BUGS
 share/doc/wl/en/NEWS
 share/doc/wl/en/README.md
--- misc/emacs-wiki/PLIST.orig
+++ misc/emacs-wiki/PLIST
@@ -1,5 +1,5 @@
 @comment $NetBSD: PLIST,v 1.3 2011/02/13 09:59:34 ryoon Exp $
-info/emacs-wiki
+${EMACS_INFOPREFIX}/emacs-wiki
 ${EMACS_LISPPREFIX}/emacs-wiki/contrib/cgi.el
 ${EMACS_LISPPREFIX}/emacs-wiki/contrib/htmlize.el
 ${EMACS_LISPPREFIX}/emacs-wiki/contrib/httpd.el
--- print/auctex/PLIST.orig
+++ print/auctex/PLIST
@@ -1,6 +1,6 @@
 @comment $NetBSD: PLIST,v 1.21 2024/05/01 20:36:20 wiz Exp $
-info/auctex.info
-info/preview-latex.info
+${EMACS_INFOPREFIX}/auctex.info
+${EMACS_INFOPREFIX}/preview-latex.info
 share/doc/auctex/tex-ref.pdf
 ${EMACS_LISPPREFIX}/auctex.el
 ${EMACS_LISPPREFIX}/auctex/bib-cite.el
--- www/emacs-w3m/PLIST.orig
+++ www/emacs-w3m/PLIST
@@ -1,6 +1,6 @@
 @comment $NetBSD: PLIST,v 1.16 2024/07/02 13:13:25 mef Exp $
-info/emacs-w3m-ja.info.gz
-info/emacs-w3m.info.gz
+${EMACS_INFOPREFIX}/emacs-w3m-ja.info.gz
+${EMACS_INFOPREFIX}/emacs-w3m.info.gz
 share/emacs-w3m/COPYING
 share/emacs-w3m/ChangeLog
 share/emacs-w3m/ChangeLog.1
--- lang/bigloo/PLIST.orig
+++ lang/bigloo/PLIST
@@ -19,7 +19,7 @@
 bin/bigloo.sh
 bin/bigloo${PKGVERSION}
 ${PLIST.emacs}bin/cigloo
-info/bigloo.info
+${EMACS_INFOPREFIX}/bigloo.info
 lib/bigloo/${PKGVERSION}/Makefile.api
 lib/bigloo/${PKGVERSION}/Makefile.api-safe
 lib/bigloo/${PKGVERSION}/Makefile.config
