--- editors/emacs29/buildlink3.mk.orig
+++ editors/emacs29/buildlink3.mk
@@ -7,7 +7,7 @@
 EMACS_BUILDLINK3_MK:=
 
 .include "../../editors/emacs/modules.mk"
-BUILDLINK_API_DEPENDS.emacs+=	${_EMACS_REQD}
+BUILDLINK_API_DEPENDS.emacs+=	${_EMACS_REQD_ANY}
 BUILDLINK_ABI_DEPENDS.emacs?=	emacs29>=29.1nb2
 BUILDLINK_PKGSRCDIR.emacs?=	${_EMACS_PKGDIR}
 
--- editors/emacs29-nox11/buildlink3.mk.orig
+++ editors/emacs29-nox11/buildlink3.mk
@@ -7,7 +7,7 @@
 EMACS_NOX11_BUILDLINK3_MK:=
 
 .include "../../editors/emacs/modules.mk"
-BUILDLINK_API_DEPENDS.emacs-nox11+=	${_EMACS_REQD}
+BUILDLINK_API_DEPENDS.emacs-nox11+=	${_EMACS_REQD_ANY}
 BUILDLINK_PKGSRCDIR.emacs-nox11?=	${_EMACS_PKGDIR}
 
 BUILDLINK_CONTENTS_FILTER.emacs-nox11=	${EGREP} '.*\.el$$|.*\.elc$$'
--- editors/emacs30/buildlink3.mk.orig
+++ editors/emacs30/buildlink3.mk
@@ -7,7 +7,7 @@
 EMACS_BUILDLINK3_MK:=
 
 .include "../../editors/emacs/modules.mk"
-BUILDLINK_API_DEPENDS.emacs+=	${_EMACS_REQD}
+BUILDLINK_API_DEPENDS.emacs+=	${_EMACS_REQD_ANY}
 BUILDLINK_ABI_DEPENDS.emacs?=	emacs30>=30.1
 BUILDLINK_PKGSRCDIR.emacs?=	${_EMACS_PKGDIR}
 
--- editors/emacs30-nox11/buildlink3.mk.orig
+++ editors/emacs30-nox11/buildlink3.mk
@@ -7,7 +7,7 @@
 EMACS_NOX11_BUILDLINK3_MK:=
 
 .include "../../editors/emacs/modules.mk"
-BUILDLINK_API_DEPENDS.emacs-nox11+=	${_EMACS_REQD}
+BUILDLINK_API_DEPENDS.emacs-nox11+=	${_EMACS_REQD_ANY}
 BUILDLINK_PKGSRCDIR.emacs-nox11?=	${_EMACS_PKGDIR}
 
 BUILDLINK_CONTENTS_FILTER.emacs-nox11=	${EGREP} '.*\.el$$|.*\.elc$$'
--- editors/emacs31/buildlink3.mk.orig
+++ editors/emacs31/buildlink3.mk
@@ -7,7 +7,7 @@
 EMACS_BUILDLINK3_MK:=
 
 .include "../../editors/emacs/modules.mk"
-BUILDLINK_API_DEPENDS.emacs+=	${_EMACS_REQD}
+BUILDLINK_API_DEPENDS.emacs+=	${_EMACS_REQD_ANY}
 BUILDLINK_ABI_DEPENDS.emacs?=	emacs31>=31.1
 BUILDLINK_PKGSRCDIR.emacs?=	${_EMACS_PKGDIR}
 
--- editors/emacs31-nox11/buildlink3.mk.orig
+++ editors/emacs31-nox11/buildlink3.mk
@@ -7,7 +7,7 @@
 EMACS_NOX11_BUILDLINK3_MK:=
 
 .include "../../editors/emacs/modules.mk"
-BUILDLINK_API_DEPENDS.emacs-nox11+=	${_EMACS_REQD}
+BUILDLINK_API_DEPENDS.emacs-nox11+=	${_EMACS_REQD_ANY}
 BUILDLINK_PKGSRCDIR.emacs-nox11?=	${_EMACS_PKGDIR}
 
 BUILDLINK_CONTENTS_FILTER.emacs-nox11=	${EGREP} '.*\.el$$|.*\.elc$$'
