diff -ruN -x CVS -x work -x '*.orig' -x '*.rej' -x '._*' /usr/pkgsrc/textproc/xslide/Makefile textproc/xslide/Makefile
--- /usr/pkgsrc/textproc/xslide/Makefile	2026-09-25 09:48:13.000000000 +0000
+++ textproc/xslide/Makefile	2026-10-03 01:02:27.586527601 +0000
@@ -3,7 +3,7 @@
 
 DISTNAME=		xslide-0.2.2
 PKGNAME=		${EMACS_PKGNAME_PREFIX}${DISTNAME}
-PKGREVISION=		4
+PKGREVISION=		5
 CATEGORIES=		textproc
 MASTER_SITES=		${MASTER_SITE_SOURCEFORGE:=xslide/}
 
@@ -35,8 +35,8 @@
 do-install:
 	${INSTALL_DATA_DIR} ${DESTDIR}${EMACS_LISPPREFIX}/${PKGBASE:C|^${EMACS_PKGNAME_PREFIX}||}
 	cd ${WRKSRC} && ${INSTALL_DATA} ${SRCS} ${SRCS:.el=.elc} ${DESTDIR}${EMACS_LISPPREFIX}/${PKGBASE:C|^${EMACS_PKGNAME_PREFIX}||}
-	${INSTALL_DATA_DIR} ${DESTDIR}${PREFIX}/share/doc/xslide
-	${INSTALL_DATA} ${WRKSRC}/README.TXT ${DESTDIR}${PREFIX}/share/doc/xslide
-	${INSTALL_DATA} ${WRKSRC}/dot_emacs ${DESTDIR}${PREFIX}/share/doc/xslide
+	${INSTALL_DATA_DIR} ${DESTDIR}${PREFIX}/share/doc/${EMACS_PKGNAME_PREFIX}xslide
+	${INSTALL_DATA} ${WRKSRC}/README.TXT ${DESTDIR}${PREFIX}/share/doc/${EMACS_PKGNAME_PREFIX}xslide
+	${INSTALL_DATA} ${WRKSRC}/dot_emacs ${DESTDIR}${PREFIX}/share/doc/${EMACS_PKGNAME_PREFIX}xslide
 
 .include "../../mk/bsd.pkg.mk"
diff -ruN -x CVS -x work -x '*.orig' -x '*.rej' -x '._*' /usr/pkgsrc/textproc/xslide/PLIST textproc/xslide/PLIST
--- /usr/pkgsrc/textproc/xslide/PLIST	2009-06-14 18:17:35.000000000 +0000
+++ textproc/xslide/PLIST	2026-10-03 01:01:01.000000000 +0000
@@ -9,5 +9,5 @@
 ${EMACS_LISPPREFIX}/xslide/xslide-process.elc
 ${EMACS_LISPPREFIX}/xslide/xslide.el
 ${EMACS_LISPPREFIX}/xslide/xslide.elc
-share/doc/xslide/README.TXT
-share/doc/xslide/dot_emacs
+share/doc/${EMACS_PKGNAME_PREFIX}xslide/README.TXT
+share/doc/${EMACS_PKGNAME_PREFIX}xslide/dot_emacs
