diff -ruN -x CVS -x work -x '*.orig' -x '*.rej' -x '._*' /usr/pkgsrc/textproc/dictem/Makefile textproc/dictem/Makefile
--- /usr/pkgsrc/textproc/dictem/Makefile	2026-09-25 09:48:12.000000000 +0000
+++ textproc/dictem/Makefile	2026-10-03 01:02:27.600178060 +0000
@@ -2,7 +2,7 @@
 
 DISTNAME=		dictem-1.0.4
 PKGNAME=		${EMACS_PKGNAME_PREFIX}${DISTNAME}
-PKGREVISION=		3
+PKGREVISION=		4
 CATEGORIES=		textproc editors
 MASTER_SITES=		${MASTER_SITE_SOURCEFORGE:=dictem/}
 
@@ -16,7 +16,7 @@
 SRCS=			dictem.el
 DOCS=			README NEWS COPYING # ChangeLog
 
-INSTALLATION_DIRS+=	share/doc/dictem ${EMACS_LISPPREFIX}
+INSTALLATION_DIRS+=	share/doc/${EMACS_PKGNAME_PREFIX}dictem ${EMACS_LISPPREFIX}
 
 do-build:
 .for f in ${SRCS}
@@ -29,7 +29,7 @@
 	cd ${WRKSRC} && ${INSTALL_DATA} ${SRCS} ${SRCS:.el=.elc}	\
 			${DESTDIR}${EMACS_LISPPREFIX};		\
 	cd ${WRKSRC} && ${INSTALL_DATA} ${DOCS}				\
-			${DESTDIR}${PREFIX}/share/doc/dictem
+			${DESTDIR}${PREFIX}/share/doc/${EMACS_PKGNAME_PREFIX}dictem
 
 .include "../../editors/emacs/modules.mk"
 .include "../../mk/bsd.pkg.mk"
diff -ruN -x CVS -x work -x '*.orig' -x '*.rej' -x '._*' /usr/pkgsrc/textproc/dictem/PLIST textproc/dictem/PLIST
--- /usr/pkgsrc/textproc/dictem/PLIST	2012-07-22 11:02:40.000000000 +0000
+++ textproc/dictem/PLIST	2026-10-03 01:01:00.000000000 +0000
@@ -1,6 +1,6 @@
 @comment $NetBSD: PLIST,v 1.4 2012/07/22 11:02:40 cheusov Exp $
-share/doc/dictem/COPYING
-share/doc/dictem/NEWS
-share/doc/dictem/README
+share/doc/${EMACS_PKGNAME_PREFIX}dictem/COPYING
+share/doc/${EMACS_PKGNAME_PREFIX}dictem/NEWS
+share/doc/${EMACS_PKGNAME_PREFIX}dictem/README
 ${EMACS_LISPPREFIX}/dictem.el
 ${EMACS_LISPPREFIX}/dictem.elc
