diff -ruN -x CVS -x work -x '*.zkdoc' -x '*.zkbak' /usr/pkgsrc/textproc/dictem/Makefile textproc/dictem/Makefile
--- /usr/pkgsrc/textproc/dictem/Makefile	2024-06-16 07:45:52.000000000 +0000
+++ textproc/dictem/Makefile	2026-09-22 23:58:37.944885346 +0000
@@ -16,7 +16,7 @@
 SRCS=			dictem.el
 DOCS=			README NEWS COPYING # ChangeLog
 
-INSTALLATION_DIRS+=	share/doc/${PKGBASE} ${EMACS_LISPPREFIX}
+INSTALLATION_DIRS+=	share/doc/${EMACS_PKGNAME_PREFIX}dictem ${EMACS_LISPPREFIX}
 
 do-build:
 .for f in ${SRCS}
@@ -29,7 +29,7 @@
 	cd ${WRKSRC} && ${INSTALL_DATA} ${SRCS} ${SRCS:.el=.elc}	\
 			${DESTDIR}${EMACS_LISPPREFIX};		\
 	cd ${WRKSRC} && ${INSTALL_DATA} ${DOCS}				\
-			${DESTDIR}${PREFIX}/share/doc/${PKGBASE}
+			${DESTDIR}${PREFIX}/share/doc/${EMACS_PKGNAME_PREFIX}dictem
 
 .include "../../editors/emacs/modules.mk"
 .include "../../mk/bsd.pkg.mk"
diff -ruN -x CVS -x work -x '*.zkdoc' -x '*.zkbak' /usr/pkgsrc/textproc/dictem/PLIST textproc/dictem/PLIST
--- /usr/pkgsrc/textproc/dictem/PLIST	2012-07-22 11:02:40.000000000 +0000
+++ textproc/dictem/PLIST	2026-09-22 23:58:37.950946731 +0000
@@ -1,6 +1,6 @@
 @comment $NetBSD: PLIST,v 1.4 2012/07/22 11:02:40 cheusov Exp $
-share/doc/dictem/COPYING
-share/doc/dictem/NEWS
-share/doc/dictem/README
+share/doc/${EMACS_PKGNAME_PREFIX}dictem/COPYING
+share/doc/${EMACS_PKGNAME_PREFIX}dictem/NEWS
+share/doc/${EMACS_PKGNAME_PREFIX}dictem/README
 ${EMACS_LISPPREFIX}/dictem.el
 ${EMACS_LISPPREFIX}/dictem.elc
