diff -ruN -x CVS -x work -x '*.orig' -x '*.rej' -x '._*' /usr/pkgsrc/textproc/emacs-muse/Makefile textproc/emacs-muse/Makefile
--- /usr/pkgsrc/textproc/emacs-muse/Makefile	2026-09-25 09:48:12.000000000 +0000
+++ textproc/emacs-muse/Makefile	2026-10-03 02:12:34.457468962 +0000
@@ -2,7 +2,7 @@
 
 DISTNAME=	muse-3.20
 PKGNAME=	${EMACS_PKGNAME_PREFIX}${DISTNAME}
-PKGREVISION=	2
+PKGREVISION=	3
 CATEGORIES=	textproc editors
 MASTER_SITES=	#
 
@@ -16,7 +16,7 @@
 INFO_FILES=	yes
 
 MAKE_FLAGS+=		-I..
-INSTALLATION_DIRS=	share/doc/emacs-muse share/examples/emacs-muse
+INSTALLATION_DIRS=	share/doc/${EMACS_PKGNAME_PREFIX}emacs-muse share/examples/${EMACS_PKGNAME_PREFIX}emacs-muse
 
 EMACS_VERSIONS_INCOMPATIBLE=	emacs20 xemacs214 xemacs214nox xemacs215 xemacs215nox
 
@@ -27,13 +27,13 @@
 
 post-install:
 	${INSTALL_DATA} ${WRKSRC}/etc/emacs-wiki-migration.txt \
-		${DESTDIR}${PREFIX}/share/doc/emacs-muse
+		${DESTDIR}${PREFIX}/share/doc/${EMACS_PKGNAME_PREFIX}emacs-muse
 	${INSTALL_DATA} ${WRKSRC}/etc/muse.rnc \
-		${DESTDIR}${PREFIX}/share/doc/emacs-muse
+		${DESTDIR}${PREFIX}/share/doc/${EMACS_PKGNAME_PREFIX}emacs-muse
 	${INSTALL_DATA} ${WRKSRC}/examples/QuickStart.muse \
-		${DESTDIR}${PREFIX}/share/examples/emacs-muse
+		${DESTDIR}${PREFIX}/share/examples/${EMACS_PKGNAME_PREFIX}emacs-muse
 	${INSTALL_DATA} ${WRKSRC}/examples/*.png \
-		${DESTDIR}${PREFIX}/share/examples/emacs-muse
+		${DESTDIR}${PREFIX}/share/examples/${EMACS_PKGNAME_PREFIX}emacs-muse
 
 .include "../../editors/emacs/modules.mk"
 .include "../../mk/bsd.pkg.mk"
diff -ruN -x CVS -x work -x '*.orig' -x '*.rej' -x '._*' /usr/pkgsrc/textproc/emacs-muse/PLIST textproc/emacs-muse/PLIST
--- /usr/pkgsrc/textproc/emacs-muse/PLIST	2017-09-08 22:21:05.000000000 +0000
+++ textproc/emacs-muse/PLIST	2026-10-03 02:12:34.458015588 +0000
@@ -66,8 +66,8 @@
 ${EMACS_LISPPREFIX}/muse/muse-xml.elc
 ${EMACS_LISPPREFIX}/muse/muse.el
 ${EMACS_LISPPREFIX}/muse/muse.elc
-share/doc/emacs-muse/emacs-wiki-migration.txt
-share/doc/emacs-muse/muse.rnc
-share/examples/emacs-muse/QuickStart.muse
-share/examples/emacs-muse/emacs-muse.png
-share/examples/emacs-muse/muse-made-with.png
+share/doc/${EMACS_PKGNAME_PREFIX}emacs-muse/emacs-wiki-migration.txt
+share/doc/${EMACS_PKGNAME_PREFIX}emacs-muse/muse.rnc
+share/examples/${EMACS_PKGNAME_PREFIX}emacs-muse/QuickStart.muse
+share/examples/${EMACS_PKGNAME_PREFIX}emacs-muse/emacs-muse.png
+share/examples/${EMACS_PKGNAME_PREFIX}emacs-muse/muse-made-with.png
