diff -ruN -x CVS -x work /usr/pkgsrc/www/emacs-w3m/Makefile www/emacs-w3m/Makefile
--- /usr/pkgsrc/www/emacs-w3m/Makefile	2026-09-18 05:50:22.901880457 +0000
+++ www/emacs-w3m/Makefile	2026-09-21 12:06:22.302350337 +0000
@@ -1,10 +1,9 @@
 # $NetBSD: Makefile,v 1.40 2026/09/12 14:31:51 wiz Exp $
 
-DISTNAME=	emacs-w3m-1.4.5
+DISTNAME=	emacs-w3m-1.4.632
 CATEGORIES=	www
 MASTER_SITES=	${MASTER_SITE_GITHUB:=emacs-w3m/}
-GITHUB_TAG=	329cfcfc0448a45a0779ded7b51332a343d02a18
-PKGREVISION=	1
+GITHUB_TAG=	584d593763082c3b31552025b6d95b7d9427cc39
 
 MAINTAINER=	pkgsrc-users@NetBSD.org
 HOMEPAGE=	http://emacs-w3m.namazu.org/
diff -ruN -x CVS -x work /usr/pkgsrc/www/emacs-w3m/distinfo www/emacs-w3m/distinfo
--- /usr/pkgsrc/www/emacs-w3m/distinfo	2024-07-02 13:13:25.000000000 +0000
+++ www/emacs-w3m/distinfo	2026-09-21 12:06:24.124973044 +0000
@@ -1,5 +1,5 @@
 $NetBSD: distinfo,v 1.9 2024/07/02 13:13:25 mef Exp $
 
-BLAKE2s (emacs-w3m-1.4.5-329cfcfc0448a45a0779ded7b51332a343d02a18.tar.gz) = d9b637a1ccd60b8ccf9c1f3422a5959fcc5b7e5b7abdf7f7f32c7df5cea0a80f
-SHA512 (emacs-w3m-1.4.5-329cfcfc0448a45a0779ded7b51332a343d02a18.tar.gz) = 2551e4b0a1be1b1fdedc2182e2e3d8c7c5aafaba4e86e7f285997e7e6b0350acbdc54fdeec514a693513abd329d8164d026e2e8c7656a5940f81e56c7cd36ae1
-Size (emacs-w3m-1.4.5-329cfcfc0448a45a0779ded7b51332a343d02a18.tar.gz) = 1080822 bytes
+BLAKE2s (emacs-w3m-1.4.632-584d593763082c3b31552025b6d95b7d9427cc39.tar.gz) = 6940e5ebf4d1ea00b6f6992695a65d3ef033282bd2269bfc7becccd316261fe7
+SHA512 (emacs-w3m-1.4.632-584d593763082c3b31552025b6d95b7d9427cc39.tar.gz) = 0a6d6993c8a010cb9a1820e437123a77e1a27e945fd002edb09fe4a57cf6910dddc99768be8f2924629e03754c03d181e2053ca500a04af5da30b77333511a74
+Size (emacs-w3m-1.4.632-584d593763082c3b31552025b6d95b7d9427cc39.tar.gz) = 1102224 bytes
