--- misc/bbdb2/Makefile.orig
+++ misc/bbdb2/Makefile
@@ -34,8 +34,9 @@
 post-extract:
 	${RM} -rf ${WRKSRC}/lisp/*.elc
 
+CONFIGURE_ARGS+=	--with-emacs=${EMACS_BIN}
+
 .if ${EMACS_FLAVOR} == "xemacs"
-CONFIGURE_ARGS+=	--with-emacs=xemacs
 CONFIGURE_ARGS+=	--with-package-dir=${PREFIX}/lib/xemacs/site-packages
 INSTALL_TARGET=		install-pkg
 .else
--- misc/bbdb2/PLIST.orig
+++ misc/bbdb2/PLIST
@@ -1,6 +1,7 @@
 @comment $NetBSD: PLIST,v 1.1 2022/06/05 02:27:31 mef Exp $
 ${EMACS_INFOPREFIX}/bbdb.info
 ${FOR_emacs}${EMACS_LISPPREFIX}/bbdb/bbdb-autoloads.el
+${FOR_emacs}${EMACS_LISPPREFIX}/bbdb/bbdb-autoloads.elc
 ${EMACS_LISPPREFIX}/bbdb/bbdb-com.el
 ${EMACS_LISPPREFIX}/bbdb/bbdb-com.elc
 ${EMACS_LISPPREFIX}/bbdb/bbdb-ftp.el
