--- devel/cqual/Makefile.orig
+++ devel/cqual/Makefile
@@ -12,6 +12,7 @@
 MAKE_JOBS_SAFE=		no
 USE_PKGLOCALEDIR=	yes
 GNU_CONFIGURE=		yes
+INSTALL_MAKE_FLAGS+=	lispdir=${EMACS_LISPPREFIX}/cqual
 USE_TOOLS+=		gmake perl:run bison m4
 
 REPLACE_PERL+=		bin/*
--- devel/cqual/PLIST.orig
+++ devel/cqual/PLIST
@@ -15,20 +15,20 @@
 share/cqual/prelude.cq
 share/cqual/proto-noderef.cq
 share/doc/cqual/user-guide.pdf
-share/emacs/site-lisp/cqual/avltree.el
-share/emacs/site-lisp/cqual/cqual-pam.el
-share/emacs/site-lisp/cqual/elib-node.el
-share/emacs/site-lisp/cqual/pam-3.el
-share/emacs/site-lisp/cqual/pam-blob.el
-share/emacs/site-lisp/cqual/pam-copyright.el
-share/emacs/site-lisp/cqual/pam-executions.el
-share/emacs/site-lisp/cqual/pam-faces.el
-share/emacs/site-lisp/cqual/pam-hyper.el
-share/emacs/site-lisp/cqual/pam-overlays.el
-share/emacs/site-lisp/cqual/pam-util.el
-share/emacs/site-lisp/cqual/pam-vars.el
-share/emacs/site-lisp/cqual/pam-window.el
-share/emacs/site-lisp/cqual/stack-m.el
+${EMACS_LISPPREFIX}/cqual/avltree.el
+${EMACS_LISPPREFIX}/cqual/cqual-pam.el
+${EMACS_LISPPREFIX}/cqual/elib-node.el
+${EMACS_LISPPREFIX}/cqual/pam-3.el
+${EMACS_LISPPREFIX}/cqual/pam-blob.el
+${EMACS_LISPPREFIX}/cqual/pam-copyright.el
+${EMACS_LISPPREFIX}/cqual/pam-executions.el
+${EMACS_LISPPREFIX}/cqual/pam-faces.el
+${EMACS_LISPPREFIX}/cqual/pam-hyper.el
+${EMACS_LISPPREFIX}/cqual/pam-overlays.el
+${EMACS_LISPPREFIX}/cqual/pam-util.el
+${EMACS_LISPPREFIX}/cqual/pam-vars.el
+${EMACS_LISPPREFIX}/cqual/pam-window.el
+${EMACS_LISPPREFIX}/cqual/stack-m.el
 share/examples/cqual/README
 share/examples/cqual/linux-lock.c
 share/examples/cqual/linux-lock.i
