--- textproc/dictem/Makefile.orig
+++ textproc/dictem/Makefile
@@ -16,7 +16,7 @@
 SRCS=			dictem.el
 DOCS=			README NEWS COPYING # ChangeLog
 
-INSTALLATION_DIRS+=	share/doc/${PKGBASE} ${EMACS_LISPPREFIX}
+INSTALLATION_DIRS+=	share/doc/dictem ${EMACS_LISPPREFIX}
 
 do-build:
 .for f in ${SRCS}
@@ -29,7 +29,7 @@
 	cd ${WRKSRC} && ${INSTALL_DATA} ${SRCS} ${SRCS:.el=.elc}	\
 			${DESTDIR}${EMACS_LISPPREFIX};		\
 	cd ${WRKSRC} && ${INSTALL_DATA} ${DOCS}				\
-			${DESTDIR}${PREFIX}/share/doc/${PKGBASE}
+			${DESTDIR}${PREFIX}/share/doc/dictem
 
 .include "../../editors/emacs/modules.mk"
 .include "../../mk/bsd.pkg.mk"
