--- www/emacs-w3m/Makefile.orig
+++ www/emacs-w3m/Makefile
@@ -20,8 +20,9 @@
 
 HAS_CONFIGURE=		yes
 CONFIGURE_ARGS+=	--with-emacs=${EMACS_BIN:Q}
+CONFIGURE_ARGS+=	--with-lispdir=${EMACS_LISPPREFIX:Q}/w3m
 CONFIGURE_ARGS+=	--with-icondir=${ICONDIR}
-CONFIGURE_ARGS+=	--infodir=${PREFIX}/info
+CONFIGURE_ARGS+=	--infodir=${PREFIX}/${PKGINFODIR}
 BUILD_TARGET=		default
 INSTALL_TARGET=		${INSTALL_TARGET.${EMACS_FLAVOR}}
 #AKE_FLAGS=		prefix=${DESTDIR}${PREFIX} ICONDIR=${DESTDIR}${ICONDIR}
