--- mail/mailutils/PLIST.orig
+++ mail/mailutils/PLIST
@@ -244,8 +244,8 @@
 share/doc/mailutils/README
 share/doc/mailutils/THANKS
 share/doc/mailutils/TODO
-${PLIST.emacs}share/emacs/site-lisp/mailutils-mh.el
-${PLIST.emacs}share/emacs/site-lisp/mailutils-mh.elc
+${PLIST.emacs}${EMACS_LISPPREFIX}/mailutils-mh.el
+${PLIST.emacs}${EMACS_LISPPREFIX}/mailutils-mh.elc
 ${PLIST.nls}share/locale/ca/LC_MESSAGES/mailutils.mo
 ${PLIST.nls}share/locale/da/LC_MESSAGES/mailutils.mo
 ${PLIST.nls}share/locale/de/LC_MESSAGES/mailutils.mo
