--- devel/rsltc/Makefile.orig
+++ devel/rsltc/Makefile
@@ -33,11 +33,11 @@
 
 do-install:
 	${INSTALL_PROGRAM} ${WRKSRC}/rsltc ${DESTDIR}${PREFIX}/bin
-	${INSTALL_DATA_DIR} ${DESTDIR}${PREFIX}/share/emacs/site-lisp/rsl
+	${INSTALL_DATA_DIR} ${DESTDIR}${EMACS_LISPPREFIX}/rsl
 	(cd ${WRKSRC} ; for i in rsltc.el tokenise.el rslconvert.el rsl-mode.el ; do\
 		${EMACS_BIN} -batch -l path.el -f batch-byte-compile $$i ;\
-		${INSTALL_DATA} $$i ${DESTDIR}${PREFIX}/share/emacs/site-lisp/rsl ;\
-		${INSTALL_DATA} $${i}c ${DESTDIR}${PREFIX}/share/emacs/site-lisp/rsl ;\
+		${INSTALL_DATA} $$i ${DESTDIR}${EMACS_LISPPREFIX}/rsl ;\
+		${INSTALL_DATA} $${i}c ${DESTDIR}${EMACS_LISPPREFIX}/rsl ;\
 	done)
 
 .include "../../editors/emacs/modules.mk"
--- devel/rsltc/PLIST.orig
+++ devel/rsltc/PLIST
@@ -1,10 +1,10 @@
 @comment $NetBSD: PLIST,v 1.2 2009/06/14 17:49:07 joerg Exp $
 bin/rsltc
-share/emacs/site-lisp/rsl/rsl-mode.el
-share/emacs/site-lisp/rsl/rsl-mode.elc
-share/emacs/site-lisp/rsl/rslconvert.el
-share/emacs/site-lisp/rsl/rslconvert.elc
-share/emacs/site-lisp/rsl/rsltc.el
-share/emacs/site-lisp/rsl/rsltc.elc
-share/emacs/site-lisp/rsl/tokenise.el
-share/emacs/site-lisp/rsl/tokenise.elc
+${EMACS_LISPPREFIX}/rsl/rsl-mode.el
+${EMACS_LISPPREFIX}/rsl/rsl-mode.elc
+${EMACS_LISPPREFIX}/rsl/rslconvert.el
+${EMACS_LISPPREFIX}/rsl/rslconvert.elc
+${EMACS_LISPPREFIX}/rsl/rsltc.el
+${EMACS_LISPPREFIX}/rsl/rsltc.elc
+${EMACS_LISPPREFIX}/rsl/tokenise.el
+${EMACS_LISPPREFIX}/rsl/tokenise.elc
