diff -u -r a/host/extension.bzl b/host/extension.bzl
--- a/host/extension.bzl	2026-08-26 09:45:31.293430867 +0000
+++ b/host/extension.bzl	2026-08-26 09:45:31.295095603 +0000
@@ -24,6 +24,8 @@
         return "osx"
     if os.startswith("freebsd"):
         return "freebsd"
+    if os.startswith("netbsd"):
+        return "netbsd"
     if os.startswith("openbsd"):
         return "openbsd"
     if os.startswith("linux"):
@@ -35,6 +37,7 @@
 def _host_platform_repo_impl(rctx):
     cpu = _translate_cpu(rctx.os.arch)
     os = _translate_os(rctx.os.name)
+    print("HOSTDETECT name=%s arch=%s -> cpu=%s os=%s" % (rctx.os.name, rctx.os.arch, cpu, os))
 
     cpu = "" if cpu == None else "  '@platforms//cpu:%s',\n" % cpu
     os = "" if os == None else "  '@platforms//os:%s',\n" % os
