diff -u -r a/toolchains/BUILD b/toolchains/BUILD
--- a/toolchains/BUILD
+++ b/toolchains/BUILD
@@ -129,6 +129,16 @@
     srcs = ["include/openbsd/jni_md.h"],
 )
 
+java_runtime_files(
+    name = "jni_md_header-dragonfly",
+    srcs = ["include/dragonfly/jni_md.h"],
+)
+
+java_runtime_files(
+    name = "jni_md_header-netbsd",
+    srcs = ["include/netbsd/jni_md.h"],
+)
+
 # The Java native interface. Depend on this package if you #include <jni.h>.
 #
 # See test_jni in third_party/bazel/src/test/shell/bazel/bazel_java_test.sh for
@@ -146,6 +156,8 @@
         "@bazel_tools//src/conditions:linux_riscv64": [":jni_md_header-linux"],
         "@bazel_tools//src/conditions:linux_s390x": [":jni_md_header-linux"],
         "@bazel_tools//src/conditions:linux_x86_64": [":jni_md_header-linux"],
+        "@bazel_tools//src/conditions:netbsd": [":jni_md_header-netbsd"],
+        "@bazel_tools//src/conditions:dragonfly": [":jni_md_header-dragonfly"],
         "@bazel_tools//src/conditions:openbsd": [":jni_md_header-openbsd"],
         "@bazel_tools//src/conditions:windows": [":jni_md_header-windows"],
         "//conditions:default": [],
@@ -159,6 +171,8 @@
         "@bazel_tools//src/conditions:linux_riscv64": ["include/linux"],
         "@bazel_tools//src/conditions:linux_s390x": ["include/linux"],
         "@bazel_tools//src/conditions:linux_x86_64": ["include/linux"],
+        "@bazel_tools//src/conditions:netbsd": ["include/netbsd"],
+        "@bazel_tools//src/conditions:dragonfly": ["include/dragonfly"],
         "@bazel_tools//src/conditions:openbsd": ["include/openbsd"],
         "@bazel_tools//src/conditions:windows": ["include/win32"],
         "//conditions:default": [],
