--- src/c-stdaux-generic.h
+++ src/c-stdaux-generic.h
@@ -31,6 +31,8 @@
  *   Compiler.
  * - ``C_COMPILER_MSVC``: The compiling software is compatible to Microsoft
  *   Visual Studio (use ``_MSC_VER`` to check for specific version support).
+ * - ``C_OS_BSD``: The target system is compatible to BSD (FreeBSD, NetBSD,
+ *   OpenBSD, or DragonFly BSD).
  * - ``C_OS_LINUX``: The target system is compatible to Linux.
  * - ``C_OS_MACOS``: The target system is compatible to Apple MacOS.
  * - ``C_OS_WINDOWS``: The target system is compatible to Microsoft Windows.
@@ -58,6 +60,10 @@
 
 #if defined(_MSC_VER)
 #  define C_COMPILER_MSVC 1
+#endif
+
+#if defined(__FreeBSD__) || defined(__NetBSD__) || defined(__OpenBSD__) || defined(__DragonFly__)
+#  define C_OS_BSD 1
 #endif
 
 #if defined(__linux__)
--- src/c-stdaux.h
+++ src/c-stdaux.h
@@ -46,7 +46,7 @@
 #  include <c-stdaux-gnuc.h>
 #endif
 
-#if defined(C_OS_LINUX) || defined(C_OS_MACOS)
+#if defined(C_OS_BSD) || defined(C_OS_LINUX) || defined(C_OS_MACOS)
 #  include <c-stdaux-unix.h>
 #endif
 
--- src/test-api.c
+++ src/test-api.c
@@ -36,6 +36,9 @@
 
         /* C_OS_* */
         {
+#if defined(__FreeBSD__) || defined(__NetBSD__) || defined(__OpenBSD__) || defined(__DragonFly__)
+                c_assert(C_OS_BSD);
+#endif
 #ifdef __linux__
                 c_assert(C_OS_LINUX);
 #endif
